mkdocs_awesome_pages_plugin/tests/navigation/test_order_and_sort.py

```diff
@@ -0,0 +1,279 @@
+from .base import NavigationTestCase
+from ...meta import Meta, MetaNavItem, MetaNavRestItem
+
+
+class TestOrderAndSort(NavigationTestCase):
+    def test_asc(self):
+        navigation = self.createAwesomeNavigation(
+            [self.page("1"), self.page("2"), self.page("3"), Meta(order=Meta.ORDER_ASC)]
+        )
+
+        self.assertNavigationEqual(navigation.items, [self.page("1"), self.page("2"), self.page("3")])
+        self.assertValidNavigation(navigation.to_mkdocs())
+
+    def test_asc_natural(self):
+        navigation = self.createAwesomeNavigation(
+            [
+                self.page("100."),
+                self.page("20."),
+                self.page("3."),
+                Meta(order=Meta.ORDER_ASC, sort_type=Meta.SORT_NATURAL),
+            ]
+        )
+
+        self.assertNavigationEqual(navigation.items, [self.page("3."), self.page("20."), self.page("100.")])
+        self.assertValidNavigation(navigation.to_mkdocs())
+
+    def test_desc(self):
+        navigation = self.createAwesomeNavigation(
+            [
+                self.page("1"),
+                self.page("2"),
+                self.page("3"),
+                Meta(order=Meta.ORDER_DESC),
+            ]
+        )
+
+        self.assertNavigationEqual(navigation.items, [self.page("3"), self.page("2"), self.page("1")])
+        self.assertValidNavigation(navigation.to_mkdocs())
+
+    def test_desc_natural(self):
+        navigation = self.createAwesomeNavigation(
+            [
+                self.page("100."),
+                self.page("20."),
+                self.page("3."),
+                Meta(order=Meta.ORDER_DESC, sort_type=Meta.SORT_NATURAL),
+            ]
+        )
+
+        self.assertNavigationEqual(navigation.items, [self.page("100."), self.page("20."), self.page("3.")])
+        self.assertValidNavigation(navigation.to_mkdocs())
+
+    def test_section_asc(self):
+        navigation = self.createAwesomeNavigation(
+            [
+                self.page("1"),
+                self.page("3"),
+                self.section("2", [self.page("a", "2/a.md")], "2"),
+                Meta(order=Meta.ORDER_ASC),
+            ]
+        )
+
+        self.assertNavigationEqual(
+            navigation.items,
+            [
+                self.page("1"),
+                self.section("2", [self.page("a", "2/a.md")], "2"),
+                self.page("3"),
+            ],
+        )
+        self.assertValidNavigation(navigation.to_mkdocs())
+
+    def test_section_asc_natural(self):
+        navigation = self.createAwesomeNavigation(
+            [
+                self.page("1"),
```

Comment:
Did you intentionally not use 100, 20 and 3 form here on?

Reply:
Oops. Fixed it for the `SORT_NATURAL` tests.